### DISCUSSION TITLE
Documentation about RStudio Connect + targets

### DISCUSSION THREAD

**Discussion by ian-flores** [ropensci/targets#745] [Show and tell]
Hi Will, 

I have written an article about deploying a targets pipeline to RStudio Connect. It's hosted here: https://solutions.rstudio.com/r/workflows/ However, I thought it made sense to link it from the package documentation as a way of documenting a pattern of deployment. What do you think would be the best format for that? Maybe a vignette?

**Comment by wlandau** [comment]
That's awesome, Ian! Thanks so much for your work on this!

I just pushed https://github.com/ropensci/targets/commit/113b530988a4b52d9a04efee418ab5a97f37c5d7 to link https://solutions.rstudio.com/r/workflows/ and https://github.com/sol-eng/targets-deployment-rsc from the README. And if you find a good place to mention it in https://github.com/ropensci-books/targets, I would welcome PRs. I try to keep `targets` itself light on vignettes, relying on the manual as much as possible.

**Comment by TylerGrantSmith** [comment]
This is good stuff!  It doesn't exactly fit my current use case, but it is good to know about this functionality. 
 Currently, my pipelines need access to data that the is only available locally.  I have the pipeline run on my local machine and then at the end there are targets that write the final dataset to a `pins` board located in Connect and publishes an rmarkdown document with all of the details.  I then access this data (via `pins`) from a shiny app.  @ian-flores is there a better way I should do this?
